tests/python/test_cg.py

```diff
@@ -0,0 +1,39 @@
+import numpy as np
+import pytest
+
+import taichi as ti
+from tests import test_utils
+
+
+@pytest.mark.parametrize("ti_dtype", [ti.f32, ti.f64])
+@test_utils.test(arch=ti.cpu)
```

Comment:
Shall we include a test for CUDA backend here, too?

Reply:
CUDA backend CG solver hasn't been implemented yet.